sdcm/ec2_client.py

```diff
@@ -133,6 +134,10 @@ def _request_spot_fleet(self, instance_type, image_id, region_name, network_if,
         LOGGER.debug('Spot fleet request: %s', request_id)
         return request_id
 
+    def _get_cloud_init_config(self):
+        cloud_init = Path("./sdcm/provision/aws-cloud-init.txt")
```

Comment:
We should not use relative file paths if possible.
Use `sct_abs_path` utility function [1] from SCT in such cases.

[1] https://github.com/scylladb/scylla-cluster-tests/blob/fa8df823/sdcm/sct_events/base.py#L37

Reply:
Good catch - I moved to using `sct_abs_path` here.